ops/k8s/secrets/api-secrets.yaml.ctmpl

```diff
@@ -8,7 +8,6 @@ metadata:
   name: api-secrets
   namespace: data-repo
 data:
-  drop-all-on-start: {{ .Data.dropAllOnStart | base64Encode }}
```

Comment:
Why are we removing dropAllOnStart? I think this is different than db.stairway.forceClean.

Reply:
dropAllOnStart is false in application.properties and in the api-secrets.json and it's not really a secret value so i didn't think it needed to be encoded and set as a env var. it can pick it up from props.